api/src/OpenTelemetry/Internal/Trace/Types.hs

```diff
@@ -87,7 +87,6 @@ data TracerProvider = TracerProvider
   , tracerProviderAttributeLimits :: !AttributeLimits
   , tracerProviderSpanLimits :: !SpanLimits
   , tracerProviderPropagators :: !(Propagator Context RequestHeaders ResponseHeaders)
-  , tracerProviderLogger :: LogRecord Text -> IO ()
```

Comment:
This is not the correct way to emit a `LogRecord`. This was a hypothetical implementation written before Logging was fleshed out.